src/notification.cpp

```diff
@@ -653,8 +906,13 @@ void Notification::close()
 /*!
     \qmlsignal Notification::clicked()
 
-    Sent when the notification is clicked (its default action is invoked).
```

Comment:
Not really; remote actions are a significant undertaking with installing a systemd service and registering handlers. I will add a comment, but we would need to provide a complete guide for it to be useful advice.